salt/output/highstate.py

```diff
@@ -186,13 +186,18 @@ def _format_host(host, data, indent_level=1):
     nchanges = 0
     strip_colors = __opts__.get("strip_colors", True)
 
-    if isinstance(data, int) or isinstance(data, six.string_types):
+    if data is None or data is True or data is False:
+        nchanges = 1
+        hstrs.append(('{0}    {1}{2[ENDC]}'.format(hcolor, data, colors)))
+        hcolor = colors["CYAN"]  # Print the minion name in cyan
+    elif isinstance(data, int) or isinstance(data, six.string_types):
         # Data in this format is from saltmod.function,
         # so it is always a 'change'
         nchanges = 1
-        hstrs.append(("{0}    {1}{2[ENDC]}".format(hcolor, data, colors)))
+        for data in data.splitlines():
```

Comment:
The `int` type has no attribute `splitlines`.

Reply:
Possibly you've meant to handle `int` in the first block handling `None` and `bool`?

Reply:
That was oversight on my part.
Odd, I never realized that True and False are numerically equivalent to  1 and 0 in python
So, the first check just needs to be
```if isinstance(data, int) or data is None:```

In the future, if someone wanted to apply different colors to different data types, then it might make sense to do a ```if data is True or data is False```  check. 
But, there's no need to perform that check in the current module (it's a bit wordy anyway) 

Ok, I've just pushed the new commit, this one should pass no problem. 

> could you please write a unit test for it?
I'm not sure how to do that, I'll research that a bit later.

Reply:
I'm going to close this PR and submit a new one, the current PR is too messy given the relative simplicity of the change requested.

Reply:
It's okay. Thank you for your work!

Reply:
the new PR# is 56598
The ```highstate.py``` file changed since my last pull, which is the reason for the 2nd commit.